addons/point_of_sale/point_of_sale_view.xml

```diff
@@ -36,6 +36,7 @@
                                     <field name="qty" on_change="onchange_qty(product_id, discount, qty, price_unit, context)"/>
                                     <field name="price_unit" on_change="onchange_qty(product_id, discount, qty, price_unit, context)"/>
                                     <field name="discount"  on_change="onchange_qty(product_id, discount, qty, price_unit, context)"/>
+                                    <field name="tax_ids"/>
```

Comment:
I guess it's supposed to be readonly but it doesn't hurt.. maybe put it at the model level just in case?